deps/tls/common.lua

```diff
@@ -26,9 +26,59 @@ local uv = require('uv')
 local utils = require('utils')
 
 local createCredentials
-local DEFAULT_CIPHERS = 'ECDHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' .. -- TLS 1.2
-                        'RC4:HIGH:!MD5:!aNULL:!EDH'                     -- TLS 1.0
-local DEFAULT_SECUREPROTOCOL = 'TLSv1_2'
+local DEFAULT_CIPHERS
+local DEFAULT_SECUREPROTOCOL
+
+local isLibreSSL = function()
+  local _, _, V = openssl.version()
+  return V:find('^LibreSSL')
+end
+
+local isTLSv1_3 = function()
+  if isLibreSSL() then
+    return false
+  end
+
+  local _, _, V = openssl.version(true)
+  return V > 0x10100000
+end
+
+DEFAULT_CIPHERS = 'ECDHE-RSA-AES128-SHA256:AES128-GCM-SHA256:' ..     --TLS 1.2
```

Comment:
Did you intentionally remove the TLS 1.3 cipher when you force pushed the new commit?

Reply:
When I try to use tls1_3 secureprotocol,  report ssl error 'no share cipher', after test ok, I'll set right cipher back

Reply:
When I try to use tls1_3 secureprotocol,  report ssl error 'no share cipher', after test ok, I'll set right cipher back